hosted-utils-api-server/src/routes/browser-circuit.mjs

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import AWS from 'aws-sdk';
+import config from 'config';
+
+const { accessKeyId, secretAccessKey } = config.AWS;
+AWS.config.update({ accessKeyId, secretAccessKey });
+const s3 = new AWS.S3();
+
+const myBucket = 'nightfallv3';
```

Comment:
separate out the buckets from which to get the circuits + abis and the proving keys. Make them environmental variables as the stub and non-stub varieties might be stored elsewhere

Reply:
done